lib/sidekiq/middleware/server/datadog.rb

```diff
@@ -41,7 +41,10 @@ def initialize(opts={})
         end
 
         def call(worker, job, queue, *)
-          start = Time.now
+          start = {
```

Comment:
pls just assign 2 vars instead of a hash:
```
start = Time.now
clock = Process.clock_gettime(Process::CLOCK_MONOTONIC, :millisecond)
```

Reply:
For sure, is it because a hash is more expensive?